tools/swift-api-digester/swift-api-digester.cpp

```diff
@@ -1019,12 +1019,30 @@ static bool isFunctionTypeNoEscape(Type Ty) {
   return false;
 }
 
+/// Converts a DeclBaseName to a string by assigning special names strings and
+/// escaping identifiers that would clash with these strings using '`'
+static StringRef getEscapedName(DeclBaseName name) {
+  switch (name.getKind()) {
+  case DeclBaseName::Kind::Subscript:
+    return "subscript";
+  case DeclBaseName::Kind::Normal:
+    return llvm::StringSwitch<StringRef>(name.getIdentifier().str())
```

Comment:
I don't think that makes sense. You wouldn't want to see `func func` in the output of this tool. @nkcsgexi, what do you think? (Sorry to pull you in mid-discusson.)